### ISSUE TITLE
Add Wait delays to Dynamic conditions

### ISSUE THREAD

**Issue by SamuelPage99** [zaafar/GameOverlay#230]
I often use expressions such as ` MonsterCount(MonsterRarity.Rare)>0` for dynamic expressions. It works well but there are times where I wish I could add a Wait parameter on a dynamic rule.

Would that be possible?

**Comment by zaafar** [comment]
Yeah, agree, like all conditions, dynamic condition should also allow components such as wait component.
